testsuite/tests/typing-core-bugs/const_int_hint.ml

```diff
@@ -2,11 +2,11 @@
    * expect
 *)
 
-let _ = Int32.(add 1 2l);;
+let _ = Int32.add 1 2l;;
```

Comment:
In passing, this isn’t a necessary change, right?

Reply:
That's correct. Happy to revert if you think this change does more harm (changing this line's Git history) than good (more consistent formatting).

Reply:
All module uses in this test file were consistently using a local open before 🙂

Reply:
Fair point :) I've undone these changes.